connectors/DBF2XML/apiDefinition.swagger.json

```diff
@@ -0,0 +1,95 @@
+{
+  "basePath": "/",
+  "consumes": [
+    "application/json"
+  ],
+  "definitions": {},
+  "host": "dbf2xml.azurewebsites.net",
+  "info": {
+    "description": "A service that allows content to be converted from dbf format to another.",
+    "title": "DBF2XML",
+    "version": "1.0.0",
+    "x-ms-api-annotation": {
+      "status": "Preview"
+    }
+  },
+  "parameters": {},
+  "paths": {
+    "/api/DBF2XML": {
+      "post": {
+        "description": "Convert base64 content db file to xml file.",
+        "operationId": "Dbf2XmlConvert",
+        "parameters": [
+          {
+            "default": "ZeBVvhUSY/fpGA2uJTOKvIRTYkNXNQEl2TaHJO9Wq39wQB8ZXdYPWA==",
```

Comment:
This string looks like a secret. Although it can be just a base64 file. Please confirm this isn't secret and please remove if it is a secret.

Reply:
Hi, it isn't a secret string. This string is Azure Function path for call https://dbf2xml.azurewebsites.net/api/DBF2XML?code=ZeBVvhUSY/fpGA2uJTOKvIRTYkNXNQEl2TaHJO9Wq39wQB8ZXdYPWA==